ipatests/test_webui/test_selinuxusermap.py

```diff
@@ -89,9 +99,22 @@ def test_mod(self):
 
         self.mod_rule_tables(tables, categories, [])
 
+        # add associations then cancel
+        def get_t_vals(t):
+            table = t[0]
+            k = t[1]
+            e = []
+            if len(t) > 2:
+                e = t[2]
+            return table, k, e
```

Comment:
This helper method seems redundant to me. Can be solved just by:

```
for t in tables:
     table = t[0]
     keys = t[1]        
     self.add_table_associations(table, [keys[0]], confirm_btn='cancel')
```

I also wonder why the same method in `mod_rule_tables` has the `e` in it. But that's another story.

Reply:
Fixed.